Calculator.c

```diff
@@ -0,0 +1,101 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+
+int main()
+{
+    char s[100];
+    printf("Enter the expression for evaluation: ");
+    scanf("%[^\n]",s);
+
+    int arr_num[100],index_num=0,index_op=0;
+    char arr_op[100];
+    int digit=0;
+    int i,j,k;
```

Comment:
what is i, j , k? Give proper naming to variables, as shared earlier at all places.